engine/apps/slack/models/slack_message.py

```diff
@@ -105,7 +110,11 @@ def permalink(self) -> typing.Optional[str]:
 
         try:
             result = SlackClient(self.slack_team_identity).chat_getPermalink(
-                channel=self.channel.slack_id, message_ts=self.slack_id
+                # TODO: once _channel_id has been fully migrated to channel, remove _channel_id
+                # see https://raintank-corp.slack.com/archives/C06K1MQ07GS/p173255546
+                # channel=self.channel.slack_id,
+                channel=self._channel_id,
```

Comment:
"fix" from https://github.com/grafana/oncall/pull/5292 (let's wait to read from `channel` until the data-migration mentioned in that PR has been fully finished)